tiup/tiup-component-dm-prune.md

```diff
@@ -0,0 +1,25 @@
+---
+title: tiup dm prune
+---
+
+# tiup dm prune
+
+When [scaling in the cluster]<!--(/tiup/tiup-component-dm-scale-in.md) -->, a small amount of metadata in etcd is not cleaned up, which usually causes no problem. If you need to clean up the metadata, you can manually execute the `tiup cluster prune` command.
```

Comment:
Should the `tiup cluster prune` here be `tiup dm prune`? Because this file introduces the `tiup dm prune` command...

Also, the Chinese version might need to be edited as well.

Reply:
updated